client/src/pages/Result.js

```diff
@@ -1,104 +1,118 @@
+`;
+
+const Tags = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+  flex-direction: row;
+  margin-bottom: 2em;
 `;
 
 const Description = styled.div`
-  padding-left: 1em;
+  justify-content: center;
 
   img {
     height: 1.2em;
     padding-right: 0.3em;
     margin-bottom: -0.2em;
   }
-  h3,
   p {
     padding-bottom: 0.5em;
-  }
-  p {
     font-size: 0.9em;
   }
 `;
 
+const Location = styled.div`
+  margin-bottom: 2em;
+`;
+
 export function Result() {
-  const [results, setResults] = useState([]);
+  const [offers, setOffers] = useState(null);
   const [isLoading, setIsLoading] = useState(true);
   //const [query, setQuery] = useState("");
 
   useEffect(() => {
     async function fetchData() {
       setIsLoading(true);
-      const offers = await fetchResults();
-      setResults(offers);
+      const offers = await fetchOffer();
+      setOffers(offers);
       setIsLoading(false);
     }
     fetchData();
   }, []);
 
-  if (isLoading || results === null) {
-    //return <LoadingScreen />;
-    console.log("loading screen");
+  if (isLoading || offers === null) {
+    return <div>loading...</div>;
   }
 
-  /*
-  const filteredResults = results.filter((result) => {
-    return result.title.toLowerCase().match(query.toLowerCase());
-  });*/
-
   return (
     <>
-      <GlobalStyles />
-      <BrowserRouter>
-        <SearchHeader />
-        <Main>
-          <List>
-            {results.map((result) => (
-              <ListItem key={result.id} href={`/offers/${result.id}`}>
-                <CategoryImage src={fruitsSrc} alt="offer title" />
-                <Description>
-                  <h3>{result.title}</h3>
-                  <p>
-                    <img src={LocationSrc} alt="locationpicker icon" />
-                    {result.location}
-                  </p>
-                  <p>
-                    <img src={DateSrc} alt="calendar icon" />
-                    {result.pickup_date}&nbsp;
-                    <img src={TimeSrc} alt="clock icon" />
-                    {result.pickup_time}&nbsp;Uhr
-                  </p>
-                </Description>
-              </ListItem>
+      <BreadcrumbHeader />
+      <Main>
+        <ListItem key={offers.id}>
+          <CategoryImage src={fruitsSrc} alt="offer title" />
+          <Title>
+            <h1>{offers.title}</h1>
+            <h2>{offers.category}</h2>
+          </Title>
+          <Tags>
+            {offers.tags.map((tag) => (
+              <Tag key={tag}>{`${tag}`}</Tag>
             ))}
-          </List>
-        </Main>
-        <Footer />
-      </BrowserRouter>
+          </Tags>
+          <Description>
+            <Location>
+              <p>
+                <img src={LocationSrc} alt="locationpicker icon" />
+                {offers.location.street}
+              </p>
+              <p>&nbsp;&nbsp;&nbsp;&nbsp;{offers.location.zip}</p>
+              <p>&nbsp;&nbsp;&nbsp;&nbsp;{offers.location.city}</p>
```

Comment:
You could use `:nth-child` to give these two `p` tags a specific margin or padding, so that you don't have to use all the `&nbsp`

Reply:
Perfect, thank you! Will try!